metaci/testresults/tests/test_testresults_views.py

```diff
@@ -0,0 +1,117 @@
+import pytest
```

Comment:
I would call this file test_views.py. I try to follow the pattern that if the actual code is in somepackage/somemodule.py, then the corresponding tests are in somepackage/tests/test_somemodule.py

(Oddbird goes even further and puts them in somepackage/tests/somemodule.py without the test_ prefix, but then it's trickier to make pytest find the right files so I have mixed feelings about it.)

Reply:
Sounds good to me.